styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
+  border: 0;
+  border-top: 1px solid $gray-lighter;
+  margin-bottom: $line-height-computed / 4;
+  margin-top:    $line-height-computed / 4;
```

Comment:
Then we need to define a class for that and go back to the strategy of people updating the CMS. We're trying to get away from styling tags, not continuing doing it.